kiwi/xml_description.py

```diff
@@ -52,9 +53,9 @@ class XMLDescription(object):
         path to base XML description file
 
     """
-    def __init__(self, description, derived_from=None):
+    def __init__(self, description=None, derived_from=None, content=None):
         self.description_xslt_processed = NamedTemporaryFile()
-        self.description = description
+        self.description = content and BytesIO(content) or description
```

Comment:
`self.description = BytesIO(content) if content else description`

But I found this a bit wrong that content have move priority than description when both are passed. I would drop content here